src/components/article/ImageBanner/index.js

```diff
@@ -22,4 +23,4 @@ export default class ImageBanner extends Component {
       </React.Fragment>
     );
   }
-}
+}
```

Comment:
Hey @BagzieGracious, please consider adding an empty line here

Reply:
I have added an empty line at the end